var/spack/repos/builtin/packages/neurodamus-core/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+from spack import *
+import os
+import shutil
+
+
+class NeurodamusCore(Package):
+    """Library of channels developed by Blue Brain Project, EPFL"""
+
+    homepage = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
+    git      = "ssh://bbpcode.epfl.ch/sim/neurodamus-core"
+
+    version('develop', git=git, branch='master')
+    version('2.0.0', tag='2.0.0')
```

Comment:
any specific reason to start with 2.0.0? I would start with 0.1 :)

Reply:
Available tags are:
1.3.0
1.3.1
1.3.2
1.6.0
1.8.0
1.9.0

I think it's time to move to V2

Reply:
Even though we moved (and cleaned) repo, rewriting history, there was no rebase (only condensed merges). So theoretically all tags are 100% the same.
Since last version was 1.9.0, with this important reorganization, I believe we can go 2.0.

Reply:
I see your point, clean repo, starting from scratch. but if we think about neurodamus as a product, even if we had reimplemented it from scratch it wouldn't go v0.1. Users will think it's a very old and unstable version and avoid to use it.
So my point is, if we keep the version scheme as x.x.x it should be higher than 1.9. Otherwise, with a different scheme (like yyyy.mm) we have full flexibility.

Reply:
As for the old tags... That's a good question. I brought them, even though it required some more code, so that users could go back to the tag, +/- as if they were in the original repo. 
HOWEVER, even though the branch commits are there, we removed unwanted files from the repo, like tests, and renamed directories. So I totally agree that might confuse the user.
My suggestion is that we give informative names to these tags, but keep them so the developers (us) have still those references (and maybe one day we can rebuild those old versions from the new repo... whatever). They won't hurt